fuzz/dex/v0.1/fuzzDexExecutorEventAddLiquidity.go

```diff
@@ -0,0 +1,184 @@
+	if err != nil {
+		return err
+	}
+	tokenBBefore, err := pfe.getTokens(user, swapPair.secondToken)
+	if err != nil {
+		return err
+	}
+	tokenLpBefore, err := pfe.getTokens(user, swapPair.lpToken)
+	if err != nil {
+		return err
+	}
+
+	output, err := pfe.executeTxStep(fmt.Sprintf(`
+	{
+		"step": "scCall",
+		"txId": "accept-esdt-payment",
+		"tx": {
+			"from": "%s",
+			"to": "%s",
+			"value": "0",
+			"function": "acceptEsdtPayment",
+			"esdt": {
+				"tokenIdentifier": "str:%s",
+				"value": "%d"
+			},
+			"arguments": [
+			],
+			"gasLimit": "100,000,000",
+			"gasPrice": "0"
+		},
+		"expect": {
+			"out": [],
+			"status": "0",
+			"message": "",
+			"gas": "*",
+			"refund": "*"
+		}
+	}`,
+		user,
+		swapPair.address,
+		swapPair.firstToken,
+		amountA,
+	))
+	if err != nil {
+		return err
+	}
+
+	output, err = pfe.executeTxStep(fmt.Sprintf(`
+	{
+		"step": "scCall",
+		"txId": "accept-esdt-payment",
+		"tx": {
+			"from": "%s",
+			"to": "%s",
+			"value": "0",
+			"function": "acceptEsdtPayment",
+			"esdt": {
+				"tokenIdentifier": "str:%s",
+				"value": "%d"
+			},
+			"arguments": [
+			],
+			"gasLimit": "100,000,000",
+			"gasPrice": "0"
+		},
+		"expect": {
+			"out": [],
+			"status": "0",
+			"message": "",
+			"gas": "*",
+			"refund": "*"
+		}
+	}`,
+		user,
+		swapPair.address,
+		swapPair.secondToken,
+		amountB,
+	))
+	if err != nil {
+		return err
+	}
+
+	output, err = pfe.executeTxStep(fmt.Sprintf(`
+	{
+		"step": "scCall",
+		"txId": "add-liquidity",
+		"tx": {
+			"from": "%s",
+			"to": "%s",
+			"value": "0",
+			"function": "addLiquidity",
+			"arguments": [
+				"%d",
+				"%d",
+				"%d",
+				"%d"
+			],
+			"gasLimit": "100,000,000",
+			"gasPrice": "0"
+		}
+	}`,
+		user,
+		swapPair.address,
+		amountA,
+		amountB,
+		amountAmin,
+		amountBmin,
+	))
+
+	if err == nil && output.ReturnCode == vmi.Ok {
+		statistics.addLiquidityHits += 1
+
+		rawEquivalentAfter, errAfter := pfe.querySingleResultStringAddr(pfe.ownerAddress, swapPair.address,
+			"getEquivalent", fmt.Sprintf("\"str:%s\", \"%d\"", swapPair.firstToken, 1000))
+		if errAfter != nil {
+			return errAfter
+		}
+
+		if errEquivalent == nil && !(len(rawEquivalent) == 1 && len(rawEquivalent[0]) == 0) {
+			statistics.addLiquidityPriceChecks += 1
+			if !equalMatrix(rawEquivalentAfter, rawEquivalent) {
+				return errors.New("PRICE CHANGED after add liquidity")
+			}
+		}
+
+		tokenAAfter, err := pfe.getTokens(user, swapPair.firstToken)
+		if err != nil {
+			return err
+		}
+		tokenBAfter, err := pfe.getTokens(user, swapPair.secondToken)
+		if err != nil {
+			return err
+		}
+		tokenLpAfter, err := pfe.getTokens(user, swapPair.lpToken)
+		if err != nil {
+			return err
+		}
+
+		if tokenABefore.Cmp(tokenAAfter) < 1 ||
+			tokenBBefore.Cmp(tokenBAfter) < 1 ||
+			tokenLpBefore.Cmp(tokenLpAfter) > -1 {
+			return errors.New("FAILED add liquidity balances on success")
+		}
+	} else {
+		statistics.addLiquidityMisses += 1
+
+		if output == nil {
```

Comment:
no nore need for this check if exiting quickly on err != nil

Reply:
done